packages/destination-actions/src/destinations/close/index.ts

```diff
@@ -0,0 +1,58 @@
+import type { DestinationDefinition } from '@segment/actions-core'
+import type { Settings } from './generated-types'
+
+import createUpdateContactAndLead from './createUpdateContactAndLead'
+
+const destination: DestinationDefinition<Settings> = {
+  name: 'Close',
+  slug: 'close',
+  mode: 'cloud',
+
+  authentication: {
+    scheme: 'basic',
+    fields: {
+      api_key: {
+        label: 'Api key',
+        description: 'Your Close api key.',
+        type: 'string',
+        required: true
+      },
+      contact_custom_field_id_for_user_id: {
+        label: 'Contact Custom Field ID for User ID',
+        description:
+          "Enter the ID of a Contact Custom Field that'll be " +
```

Comment:
Did we determine if there's a way to display a dropdown of them dynamically? I suppose not if they are also entering their API key at the same time here?

Reply:
Want to get it working without dynamic loading first, and add later. Not sure if it will be possible here. Hope it will be possible for the custom field mapping.